fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubDangling.java

```diff
@@ -148,23 +146,19 @@ private CompletableFuture<Boolean> scrubIndexRangeOnly(@Nonnull FDBRecordStore s
         validateOrThrowEx(IndexTypes.VALUE.equals(index.getType()) || scrubbingPolicy.ignoreIndexTypeCheck(), "scrubbed index is not a VALUE index");
         validateOrThrowEx(store.getIndexState(index) == IndexState.READABLE, "scrubbed index is not readable");
 
-        RangeSet rangeSet = new RangeSet(indexScrubIndexRangeSubspace(store, index));
```

Comment:
Sonarcloud is failing with [java:S3776](https://sonarcloud.io/organizations/foundationdb/rules?open=java%3AS3776&rule_key=java%3AS3776) -- I think suppressing that for this method, and addressing outside this PR, is the right call here.